### PR TITLE
Show last used time on device info page

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers 

(Visdiff should be able to show this change.)

### CODE REVIEW

File: shared/constants/types/flow-types.js [hunk 1]
```diff
@@ -162,6 +162,7 @@ export type DeviceDetail = {
   provisioner?: ?Device;
   provisionedAt?: ?Time;
   revokedAt?: ?Time;
+  lastUsed?: ?Time;
```
Comment (chrisnojima) [inline shared/constants/types/flow-types.js]: This file is generated. Why is this being edited wo the protocol change?

**Comment by cjb** [comment]
@chrisnojima Oops, thanks.  Backed out that line, and added a comment to the top of the generated file saying that it's auto-generated.

**Comment by chrisnojima** [comment]
👍

**Comment by cjb** [comment]
Ah!  There are no visdiff changes because this was already mocked out.  So here's a screenshot of the live changes in the smart component on prod, for completeness:

screenshot from 2016-06-06 14-33-54

**Pull request merged**